### DISCUSSION TITLE
How to download content from torrent server?

### DISCUSSION THREAD

**Discussion by vawaver** [bubuntux/nordvpn#153] [Q&A]
I started this docker on my Ubuntu server 20.04.
Everything is running well.
I added the torrent file of ubuntu server 20.04 (ISO image) for testing. I see the ISO image was downloaded successfully (100%) on the torrent server.
My problem is that I do not know how to download this ISO file to my PC.
I cannot locate this ISO file on my torrent server. I tried to locate it via SSH console but no success.
What should I do?
Thank you for your answer.

**Comment by yeahmyusernameis32characterslong** [comment]
I'd like to second this query and I am using a Synology. I have mounted the paths in the torrent container and added the User_ID so that it has access to the folders in the mounted paths, but in qBittorrent it says it cannot make the save location. The files downloaded are inaccessible.

When the container is updated the downloaded data will be lost (I assume) as it is stuck in the container itself.
